app/views/kaminari/_gap.html.erb

```diff
@@ -0,0 +1 @@
+<li class="disabled"><a href="#">...</a></li>
```

Comment:
These views are built into the wice_grid gem, so we can't just move them. Wice_grid doesn't pick them up from here. There may be a way to point this kaminari instance to find the template files inside the wice_grid gem, but I couldn't figure out how.